README.md

```diff
@@ -98,4 +98,28 @@ eZG4zLcylZXusOv3VYBJKypBLVI74buoFfrvMcV/lQrI3Yo+6V95rNYGm+2MVxIc
 iZSejbyqjUjJBAH9GHkPsiA+w1vutdd2PuPKOV05TLmV5ZM06bmLHQjMCGMiWK0G
 8qVxFvr2NWRDB3otAjxVHR/ZQA==
 -----END PRIVATE KEY-----
-```
\ No newline at end of file
+```
+
+### Error configuration:
+
+You can setup the simulator to return specific error scenarios for the Core Identity JWT and ID Token issued. Multiple error states can be enabled and these should be passed to the following environment variables as a comma separated string:
```

Comment:
Could we add a line about how to remove error scenarios if you have just set them?